.github/workflows/persistent_store_test.yml

```diff
@@ -13,6 +13,11 @@ jobs:
     steps:
       - uses: actions/checkout@v2
 
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: nightly
+          override: true
```

Comment:
This means that the persistent storage is going to be tested/built against a toolchain that is not the one use to build and test OpenSK (i.e. the one referred in the `rust-toolchain` file at the root of the repository.

Reply:
But this is already not the case no? I had to change this otherwise it was using stable. If there's a way to use `rust-toolchain` I'd like to do that. I think the issue is that `rust-toolchain` is a symlink to a gitmodule. So I see 2 solutions:
1. Clone the submodules (waste of resources but then I use the same toolchain).
2. Keep as is since any difference should be a compiler bug.

I'm more in favor of keeping as is since this is just to run tests, so there would anyway be a difference between x86 and arm.

Reply:
Ah I see, it's a workaround because the symlink is broken until you get the submodules. Makes sense then.
I'm simply worried that this may creates some discrepancies between the workflows by not using the same toolchain (e.g. 1 workflow being broken but not the other)

Reply:
Yeah, we should do something. One solution that looks OK would be:
- Don't use a symlink and have duplicated data.
- The `setup.sh` script should check that file at the root is equal (content) to the reference file (currently `libtock-rs`)
- We could also emit a warning that the `libtock-rs` and `tock` toolchains are different, but we probably can't fix it on our side.

This would fix all github worflows that don't update the submodules.

Reply:
Rather than `setup.sh` this should be part of the checks from the CI to ensure that when we bump the kernel version we also update this file.

`libtock-rs` and `tock` don't use the same toolchain IIRC. Or at least that used to be case if they fixed it.

Reply:
Right that's even better to do it in a workflow. So should I wait until we have this check and update my workflow, or should I create a bug to remember to update the workflow once we have this check?

Reply:
Create a bug and merge this PR.
I have to think about revamping the GitHub actions anyway now that we have multiple branches.

Reply:
Sounds good, created https://github.com/google/OpenSK/issues/261.